cloud/pkg/controllermanager/edgeapplication/overridemanager/envoverrider.go

```diff
@@ -216,14 +217,87 @@ func convertToEnvVar(value interface{}) (*corev1.EnvVar, error) {
 
 func convertToEnvVarSource(value map[string]interface{}) (corev1.EnvVarSource, error) {
```

Comment:
You can obtain a value using the NestedXxx function in the package k8s.io/apimachinery/pkg/apis/meta/v1/unstructured（helpers.go）